parser.go

```diff
@@ -978,6 +980,57 @@ func fullTypeName(pkgName, typeName string) string {
 	return typeName
 }
 
+// fillDefinition additionally fills fields in definition (spec.Schema)
+// TODO: If .go file contains many types, it may work for a long time
+func fillDefinition(definition *spec.Schema, file *ast.File, typeSpecDef *TypeSpecDef) {
+	if definition.Description == "" {
```

Comment:
Please move the if statement at line 957 , so the method will be called only when is necesary.
```go
if definition.Description == "" {
	fillDefinition(definition, typeSpecDef.File, typeSpecDef)
}
```